<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  






























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Publications | Bin Zhang's Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Publications">
<meta name="description" content="An engaging 1-3 sentence description of your lab.">

<meta property="og:title" content="Publications">
<meta property="og:site_title" content="Bin Zhang's Lab">
<meta property="og:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="og:url" content="https://christear.github.io">
<meta property="og:image" content="/images/share.png">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Publications">
<meta property="twitter:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="twitter:url" content="https://christear.github.io">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="/images/share.png">


  <meta property="og:type" content="website">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "WebSite",
    
    "name": "Publications",
    "description": "An engaging 1-3 sentence description of your lab.",
    "headline": "Publications",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": "https://christear.github.io"
  }
</script>

<link rel="alternate" type="application/rss+xml" href="https://christear.github.io/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Barlow:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.7.0/css/all.css" rel="stylesheet" media="none" onload="this.removeAttribute('media'); this.onload = null;">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.7.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/details.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/table-wrap.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/background.png')" data-dark="true">
  <a href="/" class="home">
    
      <span class="logo">
        
          <svg width="227" height="116" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" overflow="hidden"><defs><clippath id="clip0"><rect x="0" y="0" width="227" height="116"></rect></clippath></defs><g clip-path="url(#clip0)"><rect x="0" y="0" width="227" height="116.417" fill="#FFFFFF" fill-opacity="0"></rect><path d="M20 33.6203C20 29.4633 23.3578 26.0934 27.5 26.0934 31.6421 26.0934 35 29.4633 35 33.6203 35 37.7773 31.6421 41.1472 27.5 41.1472 23.3578 41.1472 20 37.7773 20 33.6203Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M7.99999 69.7496C7.99999 65.5926 11.3579 62.2226 15.5 62.2226 19.6421 62.2226 23 65.5926 23 69.7496 23 73.9067 19.6421 77.2766 15.5 77.2766 11.3579 77.2766 7.99999 73.9067 7.99999 69.7496Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M69 72.7604C69 68.6033 72.3578 65.2334 76.5 65.2334 80.6422 65.2334 84 68.6033 84 72.7604 84 76.9174 80.6422 80.2874 76.5 80.2874 72.3578 80.2874 69 76.9174 69 72.7604Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M109 80.7891C109 76.6321 112.358 73.2621 116.5 73.2621 120.642 73.2621 124 76.6321 124 80.7891 124 84.9462 120.642 88.3161 116.5 88.3161 112.358 88.3161 109 84.9462 109 80.7891Z" fill="#1D7EB3" fill-rule="evenodd"></path><path d="M45 100.861C45 96.704 48.3578 93.3341 52.5 93.3341 56.6421 93.3341 60 96.704 60 100.861 60 105.018 56.6421 108.388 52.5 108.388 48.3578 108.388 45 105.018 45 100.861Z" fill="#1D7EB3" fill-rule="evenodd"></path><path d="M90 15.5557C90 11.3986 93.3579 8.02873 97.5 8.02873 101.642 8.02873 105 11.3986 105 15.5557 105 19.7127 101.642 23.0826 97.5 23.0826 93.3579 23.0826 90 19.7127 90 15.5557Z" fill-rule="evenodd"></path><path d="M73.9999 24.588C73.9999 20.431 77.3578 17.061 81.5 17.061 85.6422 17.061 89 20.431 89 24.588 89 28.745 85.6422 32.1149 81.5 32.1149 77.3578 32.1149 73.9999 28.745 73.9999 24.588Z" fill-rule="evenodd"></path><path d="M53 30.1077C53 28.4449 54.567 27.097 56.5 27.097 58.433 27.097 60 28.4449 60 30.1077 60 31.7705 58.433 33.1185 56.5 33.1185 54.567 33.1185 53 31.7705 53 30.1077Z" fill="#1D7EB3" fill-rule="evenodd"></path><path d="M103 34.6239C103 32.6839 104.343 31.1113 106 31.1113 107.657 31.1113 109 32.6839 109 34.6239 109 36.5638 107.657 38.1365 106 38.1365 104.343 38.1365 103 36.5638 103 34.6239Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M85 103.37C85 101.707 86.567 100.359 88.5 100.359 90.433 100.359 92 101.707 92 103.37 92 105.033 90.433 106.381 88.5 106.381 86.567 106.381 85 105.033 85 103.37Z" fill-rule="evenodd"></path><path d="M33.5 37.6347 75.242 69.9961" stroke="#D0602C" stroke-width="3.33578" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><path d="M20.5 69.7496 71.9851 73.3004" stroke="#D0602C" stroke-width="3.33578" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><path d="M0 0 57.2331 18.068" stroke="#1D7EB3" stroke-width="3.33578" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(1 0 0 -1.00359 58.5 99.9256)"></path><path d="M0 0 16.1214 44.193" stroke="#000000" stroke-width="3.33578" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1.00359 95.6215 22.5808)"></path><path d="M82.5 24.588 113.92 74.3286" stroke="#000000" stroke-width="3.33578" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><path d="M28.5 37.6347 50.7742 94.3809" stroke="#D0602C" stroke-width="3.33578" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><text fill="#FFFFFF" font-family="Calisto MT,Calisto MT_MSFontService,sans-serif" font-weight="400" font-size="32" transform="matrix(1 0 0 1.00359 149.319 80)">BIN</text></g></svg>
        
      </span>
    
    
      <span class="title-text" data-tooltip="Home">
        
          <span class="title">Bin Zhang's Lab</span>
        
        
      </span>
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/research/" data-tooltip="Learn about our research">
          Research
        </a>
      
    
      
        <a href="/publications/" data-tooltip="Published works">
          Publications
        </a>
      
    
      
        <a href="/team/" data-tooltip="About our team">
          Team
        </a>
      
    
      
        <a href="/joinus/" data-tooltip="Join our team">
          Join us!
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Email, address, and location">
          Contact
        </a>
      
    
  </nav>
</header>

    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - filter out blank sections
-->






  
  
  

  <section class="background" data-size="page">
    <h1 id="publications">
<i class="icon fa-solid fa-microscope"></i>Publications</h1>

<p>Looking ahead, our research aims to develop interpretable biological foundation models, with a focus on antigen presentation and RNA isoform regulation at single-cell resolution. Moving beyond traditional gene-centric approaches, we explore isoform-level expression as a fundamental representation to characterize cell types, cellular states, and their spatiotemporal dynamics. This perspective enables a more precise and mechanistic understanding of cellular regulation.</p>

<p>By integrating multi-omics data, single-cell technologies, and AI-driven modeling, we seek to construct RNA-centric gene regulatory networks that address key limitations of existing biological foundation models, particularly in interpretability and regulatory resolution. Ultimately, our work aims to advance the quantitative understanding of living systems and to support the rational design of biological processes for applications in biomedicine and beyond.</p>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  <background></background>
  <dark></dark>
  <size></size>
-->

<h2 id="highlighted">Highlighted</h2>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd4d" class="citation-image" aria-label="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512202332549.png" alt="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd4d" class="citation-title">
        Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing
      </a>

      <div class="citation-authors" tabindex="0">
        Bin Zhang, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Genetics</span>
         · 
        <span class="citation-date">04 May 2023</span>
         · 
        <span class="citation-id">doi:10.1038/s41588-023-01363-5</span>
      </div>

      
        
          <div class="citation-description">
            Aberrant RNA splicing events resulting from DNA variations are common causes of genetic disorders. Two studies published in Nature Genetics independently describe methods to decipher DNA-variant-associated aberrant splicing using high-throughput RNA sequencing data.

          </div>
        

        

        
      
    </div>
  </div>
</div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  <background></background>
  <dark></dark>
  <size></size>
-->

<h2 id="all">All</h2>

<div class="search-box">
  <input type="text" class="search-input" oninput="onSearchInput(this)" placeholder="Search items on this page">
  <button disabled data-tooltip="Clear search" aria-label="clear search" onclick="onSearchClear()">
    <i class="icon fa-solid fa-magnifying-glass"></i>
  </button>
</div>

<div class="search-info"></div>

<h3 id="2025">2025</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd3m" class="citation-image" aria-label="Stabilizing a mammalian RNA thermometer confers neuroprotection in subarachnoid hemorrhage">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512261711868.png" alt="Stabilizing a mammalian RNA thermometer confers neuroprotection in subarachnoid hemorrhage" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd3m" class="citation-title">
        Stabilizing a mammalian RNA thermometer confers neuroprotection in subarachnoid hemorrhage
      </a>

      <div class="citation-authors" data-tooltip="Min Zhang, Bin Zhang, Chengli Liu, Marco Preußner, Megha Ayachit, Weiming Li, Yafei Huang, Deyi Liu, Quanwei He, Ann-Kathrin Emmerichs, Stefan Meinke, Shu Chen, Lin Wang, Liduan Zheng, Qiubai Li, Qin Huang, Tom Haltenhof, Ruoxi Gao, Xianan Qin, Aifang Cheng, Tianzi Wei, Li Yu, Mario Schubert, Xin Gao, Mingchang Li, Florian Heyd" tabindex="0">
        Min Zhang, Bin Zhang, Chengli Liu, Marco Preußner, Megha Ayachit, …, Li Yu, Mario Schubert, Xin Gao, Mingchang Li, Florian Heyd

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Communications</span>
         · 
        <span class="citation-date">18 Sep 2025</span>
         · 
        <span class="citation-id">doi:10.1038/s41467-025-63911-3</span>
      </div>

      
        
          <div class="citation-description">
            Mammals tightly regulate their core body temperature, yet how cells sense and respond to small temperature changes remains incompletely understood. Here, we discover RNA G-quadruplexes (rG4s) as key thermosensors enriched near splice sites of cold-repressed exons. These thermosensing RNA structures, when stabilized, mask splice sites, reducing exon inclusion.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2024">2024</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g8qqnw" class="citation-image" aria-label="An AI Agent for Fully Automated Multi Omic Analyses">
        <img src="https://advanced.onlinelibrary.wiley.com/cms/asset/e7932ca7-a507-4ccc-97ce-2fbc2a59533a/advs9394-fig-0001-m.jpg" alt="An AI Agent for Fully Automated Multi Omic Analyses" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g8qqnw" class="citation-title">
        An AI Agent for Fully Automated Multi‐Omic Analyses
      </a>

      <div class="citation-authors" data-tooltip="Juexiao Zhou, Bin Zhang, Guowei Li, Xiuying Chen, Haoyang Li, Xiaopeng Xu, Siyuan Chen, Wenjia He, Chencheng Xu, Liwei Liu, Xin Gao" tabindex="0">
        Juexiao Zhou, Bin Zhang, Guowei Li, Xiuying Chen, Haoyang Li, …, Siyuan Chen, Wenjia He, Chencheng Xu, Liwei Liu, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Advanced Science</span>
         · 
        <span class="citation-date">03 Oct 2024</span>
         · 
        <span class="citation-id">doi:10.1002/advs.202407094</span>
      </div>

      
        
          <div class="citation-description">
            With the fast-growing and evolving omics data, the demand for streamlined and adaptable tools to handle bioinformatics analysis continues to grow. In response to this need, Automated Bioinformatics Analysis (AutoBA) is introduced, an autonomous AI agent designed explicitly for fully automated multi-omic analyses based on large language models (LLMs).

          </div>
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd3s" class="citation-image" aria-label="An isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512261710900.png" alt="An isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd3s" class="citation-title">
        An isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing
      </a>

      <div class="citation-authors" data-tooltip="Zhongxiao Li, Bin Zhang, Jia Jia Chan, Hossein Tabatabaeian, Qing Yun Tong, Xiao Hong Chew, Xiaonan Fan, Patrick Driguez, Charlene Chan, Faith Cheong, Shi Wang, Bei En Siew, Ian Jse-Wei Tan, Kai-Yin Lee, Bettina Lieske, Wai-Kit Cheong, Dennis Kappei, Ker-Kan Tan, Xin Gao, Yvonne Tay" tabindex="0">
        Zhongxiao Li, Bin Zhang, Jia Jia Chan, Hossein Tabatabaeian, Qing Yun Tong, …, Wai-Kit Cheong, Dennis Kappei, Ker-Kan Tan, Xin Gao, Yvonne Tay

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cell Genomics</span>
         · 
        <span class="citation-date">11 Sep 2024</span>
         · 
        <span class="citation-id">doi:10.1016/j.xgen.2024.100641</span>
      </div>

      
        
          <div class="citation-description">
            Colorectal cancer (CRC) ranks as the second leading cause of cancer deaths globally. In recent years, short-read single-cell RNA sequencing (scRNA-seq) has been instrumental in deciphering tumor heterogeneities. However, these studies only enable gene-level quantification but neglect alterations in transcript structures arising from alternative end processing or splicing. In this study, we integrated short- and long-read scRNA-seq of CRC samples to build an isoform-resolution CRC transcriptomic atlas.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2023">2023</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd4d" class="citation-image" aria-label="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512202332549.png" alt="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd4d" class="citation-title">
        Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing
      </a>

      <div class="citation-authors" tabindex="0">
        Bin Zhang, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Genetics</span>
         · 
        <span class="citation-date">04 May 2023</span>
         · 
        <span class="citation-id">doi:10.1038/s41588-023-01363-5</span>
      </div>

      
        
          <div class="citation-description">
            Aberrant RNA splicing events resulting from DNA variations are common causes of genetic disorders. Two studies published in Nature Genetics independently describe methods to decipher DNA-variant-associated aberrant splicing using high-throughput RNA sequencing data.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd36" class="citation-image" aria-label="Accurate transcriptome-wide identification and quantification of alternative polyadenylation from RNA-seq data with APAIQ">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512261715679.png" alt="Accurate transcriptome-wide identification and quantification of alternative polyadenylation from RNA-seq data with APAIQ" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd36" class="citation-title">
        Accurate transcriptome-wide identification and quantification of alternative polyadenylation from RNA-seq data with APAIQ
      </a>

      <div class="citation-authors" data-tooltip="Yongkang Long, Bin Zhang, Shuye Tian, Jia Jia Chan, Juexiao Zhou, Zhongxiao Li, Yisheng Li, Zheng An, Xingyu Liao, Yu Wang, Shiwei Sun, Ying Xu, Yvonne Tay, Wei Chen, Xin Gao" tabindex="0">
        Yongkang Long, Bin Zhang, Shuye Tian, Jia Jia Chan, Juexiao Zhou, …, Shiwei Sun, Ying Xu, Yvonne Tay, Wei Chen, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Genome Research</span>
         · 
        <span class="citation-date">28 Feb 2023</span>
         · 
        <span class="citation-id">doi:10.1101/gr.277177.122</span>
      </div>

      
        
          <div class="citation-description">
            Alternative polyadenylation (APA) enables a gene to generate multiple transcripts with different 3′ ends, which is dynamic across different cell types or conditions. Many computational methods have been developed to characterize sample-specific APA using the corresponding RNA-seq data, but suffered from high error rate on both polyadenylation site (PAS) identification and quantification of PAS usage (PAU), and bias toward 3′ untranslated regions. Here we developed a tool for APA identification and quantification (APAIQ) from RNA-seq data, which can accurately identify PAS and quantify PAU in a transcriptome-wide manner.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2022">2022</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g8p6ct" class="citation-image" aria-label="Pan-cancer pervasive upregulation of 3 UTR splicing drives tumourigenesis">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512202333121.png" alt="Pan-cancer pervasive upregulation of 3 UTR splicing drives tumourigenesis" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g8p6ct" class="citation-title">
        Pan-cancer pervasive upregulation of 3′ UTR splicing drives tumourigenesis
      </a>

      <div class="citation-authors" data-tooltip="Jia Jia Chan, Bin Zhang, Xiao Hong Chew, Adil Salhi, Zhi Hao Kwok, Chun You Lim, Ng Desi, Nagavidya Subramaniam, Angela Siemens, Tyas Kinanti, Shane Ong, Avencia Sanchez-Mejias, Phuong Thao Ly, Omer An, Raghav Sundar, Xiaonan Fan, Shi Wang, Bei En Siew, Kuok Chung Lee, Choon Seng Chong, Bettina Lieske, Wai-Kit Cheong, Yufen Goh, Wee Nih Fam, Melissa G. Ooi, Bryan T. H. Koh, Shridhar Ganpathi Iyer, Wen Huan Ling, Jianbin Chen, Boon-Koon Yoong, Rawisak Chanwat, Glenn Kunnath Bonney, Brian K. P. Goh, Weiwei Zhai, Melissa J. Fullwood, Wilson Wang, Ker-Kan Tan, Wee Joo Chng, Yock Young Dan, Jason J. Pitt, Xavier Roca, Ernesto Guccione, Leah A. Vardy, Leilei Chen, Xin Gao, Pierce K. H. Chow, Henry Yang, Yvonne Tay" tabindex="0">
        Jia Jia Chan, Bin Zhang, Xiao Hong Chew, Adil Salhi, Zhi Hao Kwok, …, Leilei Chen, Xin Gao, Pierce K. H. Chow, Henry Yang, Yvonne Tay

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Cell Biology</span>
         · 
        <span class="citation-date">26 May 2022</span>
         · 
        <span class="citation-id">doi:10.1038/s41556-022-00913-z</span>
      </div>

      
        
          <div class="citation-description">
            Most mammalian genes generate messenger RNAs with variable untranslated regions (UTRs) that are important post-transcriptional regulators. In cancer, shortening at 3′ UTR ends via alternative polyadenylation can activate oncogenes. However, internal 3′ UTR splicing remains poorly understood as splicing studies have traditionally focused on protein-coding alterations.

          </div>
        

        

        
      
    </div>
  </div>
</div>
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/background.png')" data-dark="true" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->

  <div>
    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="mailto:bin.zhang@mbzuai.ac.ae" data-tooltip="Email" data-style="bare" aria-label="Email">
      <i class="icon fa-solid fa-envelope"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://orcid.org/0000-0001-8835-8370" data-tooltip="ORCID" data-style="bare" aria-label="ORCID">
      <i class="icon fa-brands fa-orcid"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://scholar.google.com/citations?user=lFl6evoAAAAJ" data-tooltip="Google Scholar" data-style="bare" aria-label="Google Scholar">
      <i class="icon fa-brands fa-google"></i>
      
    </a>
  </div>


    
  </div>

  <div>
    © 2025
    Bin Zhang's Lab
      |   Built with
    <a href="https://github.com/greenelab/lab-website-template">
      Lab Website Template
    </a>
  </div>

  <input type="checkbox" class="dark-toggle" data-tooltip="Dark mode" aria-label="toggle dark mode" oninput="onDarkToggleChange(event)">
</footer>

  </body>
</html>
