<!DOCTYPE html>
<html lang="en" data-dark="false">
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <!--
  put your analytics (e.g. Google Analytics) tracking code here
-->

  <!--
  put your search engine verification (e.g. Google Search Console) tag here
-->

  






























<meta name="viewport" content="width=device-width, initial-scale=1">

<title>Publications | Bin Zhang's Lab</title>

<link rel="icon" href="/images/icon.png">

<meta name="title" content="Publications">
<meta name="description" content="An engaging 1-3 sentence description of your lab.">

<meta property="og:title" content="Publications">
<meta property="og:site_title" content="Bin Zhang's Lab">
<meta property="og:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="og:url" content="https://christear.github.io">
<meta property="og:image" content="/images/share.png">
<meta property="og:locale" content="en_US">

<meta property="twitter:title" content="Publications">
<meta property="twitter:description" content="An engaging 1-3 sentence description of your lab.">
<meta property="twitter:url" content="https://christear.github.io">
<meta property="twitter:card" content="summary_large_image">
<meta property="twitter:image" content="/images/share.png">


  <meta property="og:type" content="website">


<script type="application/ld+json">
  {
    "@context": "https://schema.org",
    
      "@type": "WebSite",
    
    "name": "Publications",
    "description": "An engaging 1-3 sentence description of your lab.",
    "headline": "Publications",
    "publisher": {
      "@type": "Organization",
      "logo": { "@type": "ImageObject", "url": "/images/icon.png" }
    },
    "url": "https://christear.github.io"
  }
</script>

<link rel="alternate" type="application/rss+xml" href="https://christear.github.io/feed.xml">

  <!-- Google Fonts -->
<!-- automatically get url from fonts used in theme file -->

<link rel="preconnect" href="https://fonts.gstatic.com">
<link href="https://fonts.googleapis.com/css2?display=swap&&family=Barlow:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600&amp;family=Roboto+Mono:ital,wght@0,200;0,400;0,500;0,600;1,200;1,400;1,500;1,600" rel="stylesheet">

<!-- Font Awesome icons (load asynchronously due to size) -->

<link href="https://use.fontawesome.com/releases/v6.7.0/css/all.css" rel="stylesheet" media="none" onload="this.removeAttribute('media'); this.onload = null;">
<noscript>
  <link href="https://use.fontawesome.com/releases/v6.7.0/css/all.css" rel="stylesheet">
</noscript>

  <!-- third party styles -->
<!-- https://stylishthemes.github.io/Syntax-Themes/pygments/ -->
<link href="https://cdn.jsdelivr.net/gh/StylishThemes/Syntax-Themes/pygments/css-github/pygments-tomorrow-night-eighties.css" rel="stylesheet">

<!-- include all sass in styles folder -->


  
    <link href="/_styles/-theme.css" rel="stylesheet">
  

  
    <link href="/_styles/alert.css" rel="stylesheet">
  

  
    <link href="/_styles/all.css" rel="stylesheet">
  

  
    <link href="/_styles/anchor.css" rel="stylesheet">
  

  
    <link href="/_styles/background.css" rel="stylesheet">
  

  
    <link href="/_styles/body.css" rel="stylesheet">
  

  
    <link href="/_styles/bold.css" rel="stylesheet">
  

  
    <link href="/_styles/button.css" rel="stylesheet">
  

  
    <link href="/_styles/card.css" rel="stylesheet">
  

  
    <link href="/_styles/checkbox.css" rel="stylesheet">
  

  
    <link href="/_styles/citation.css" rel="stylesheet">
  

  
    <link href="/_styles/code.css" rel="stylesheet">
  

  
    <link href="/_styles/cols.css" rel="stylesheet">
  

  
    <link href="/_styles/dark-toggle.css" rel="stylesheet">
  

  
    <link href="/_styles/details.css" rel="stylesheet">
  

  
    <link href="/_styles/feature.css" rel="stylesheet">
  

  
    <link href="/_styles/figure.css" rel="stylesheet">
  

  
    <link href="/_styles/float.css" rel="stylesheet">
  

  
    <link href="/_styles/font.css" rel="stylesheet">
  

  
    <link href="/_styles/footer.css" rel="stylesheet">
  

  
    <link href="/_styles/form.css" rel="stylesheet">
  

  
    <link href="/_styles/grid.css" rel="stylesheet">
  

  
    <link href="/_styles/header.css" rel="stylesheet">
  

  
    <link href="/_styles/heading.css" rel="stylesheet">
  

  
    <link href="/_styles/highlight.css" rel="stylesheet">
  

  
    <link href="/_styles/icon.css" rel="stylesheet">
  

  
    <link href="/_styles/image.css" rel="stylesheet">
  

  
    <link href="/_styles/link.css" rel="stylesheet">
  

  
    <link href="/_styles/list.css" rel="stylesheet">
  

  
    <link href="/_styles/main.css" rel="stylesheet">
  

  
    <link href="/_styles/paragraph.css" rel="stylesheet">
  

  
    <link href="/_styles/portrait.css" rel="stylesheet">
  

  
    <link href="/_styles/post-excerpt.css" rel="stylesheet">
  

  
    <link href="/_styles/post-info.css" rel="stylesheet">
  

  
    <link href="/_styles/post-nav.css" rel="stylesheet">
  

  
    <link href="/_styles/quote.css" rel="stylesheet">
  

  
    <link href="/_styles/rule.css" rel="stylesheet">
  

  
    <link href="/_styles/search-box.css" rel="stylesheet">
  

  
    <link href="/_styles/search-info.css" rel="stylesheet">
  

  
    <link href="/_styles/section.css" rel="stylesheet">
  

  
    <link href="/_styles/table.css" rel="stylesheet">
  

  
    <link href="/_styles/tags.css" rel="stylesheet">
  

  
    <link href="/_styles/textbox.css" rel="stylesheet">
  

  
    <link href="/_styles/tooltip.css" rel="stylesheet">
  

  
    <link href="/_styles/util.css" rel="stylesheet">
  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  

  


<!-- include all css in styles folder -->



  <!-- third party scripts -->
<script src="https://unpkg.com/@popperjs/core@2" defer></script>
<script src="https://unpkg.com/tippy.js@6" defer></script>
<script src="https://unpkg.com/mark.js@8" defer></script>

<!-- include all js in scripts folder -->


  <script src="/_scripts/anchors.js"></script>

  <script src="/_scripts/dark-mode.js"></script>

  <script src="/_scripts/fetch-tags.js"></script>

  <script src="/_scripts/search.js"></script>

  <script src="/_scripts/site-search.js"></script>

  <script src="/_scripts/table-wrap.js"></script>

  <script src="/_scripts/tooltip.js"></script>


</head>

  <body>
    







<header class="background" style="--image: url('/images/background.png')" data-dark="true">
  <a href="/" class="home">
    
      <span class="logo">
        
          <svg width="189" height="116" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" overflow="hidden"><defs><clippath id="clip0"><rect x="0" y="0" width="189" height="116"></rect></clippath></defs><g clip-path="url(#clip0)"><rect x="0" y="0" width="189" height="116.502" fill="#FFFFFF" fill-opacity="0"></rect><path d="M29 38.6667C29 35.616 31.4624 33.1429 34.5 33.1429 37.5376 33.1429 40 35.616 40 38.6667 40 41.7174 37.5376 44.1905 34.5 44.1905 31.4624 44.1905 29 41.7174 29 38.6667Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M19 66.2857C19 62.9577 21.6863 60.2597 25 60.2597 28.3137 60.2597 31 62.9577 31 66.2857 31 69.6138 28.3137 72.3117 25 72.3117 21.6863 72.3117 19 69.6138 19 66.2857Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M66 69.2987C66 65.9706 68.4624 63.2727 71.5 63.2727 74.5376 63.2727 77 65.9706 77 69.2987 77 72.6268 74.5376 75.3247 71.5 75.3247 68.4624 75.3247 66 72.6268 66 69.2987Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M97 74.8225C97 71.7718 99.4624 69.2987 102.5 69.2987 105.538 69.2987 108 71.7718 108 74.8225 108 77.8732 105.538 80.3463 102.5 80.3463 99.4624 80.3463 97 77.8732 97 74.8225Z" fill="#1D7EB3" fill-rule="evenodd"></path><path d="M48 90.3896C48 87.0616 50.4624 84.3636 53.5 84.3636 56.5376 84.3636 59 87.0616 59 90.3896 59 93.7177 56.5376 96.4156 53.5 96.4156 50.4624 96.4156 48 93.7177 48 90.3896Z" fill="#1D7EB3" fill-rule="evenodd"></path><path d="M82 25.6104C82 22.5597 84.4624 20.0866 87.5 20.0866 90.5376 20.0866 93 22.5597 93 25.6104 93 28.6611 90.5376 31.1342 87.5 31.1342 84.4624 31.1342 82 28.6611 82 25.6104Z" fill-rule="evenodd"></path><path d="M70 31.6364C70 28.5856 72.4624 26.1126 75.5 26.1126 78.5376 26.1126 81 28.5856 81 31.6364 81 34.6871 78.5376 37.1602 75.5 37.1602 72.4624 37.1602 70 34.6871 70 31.6364Z" fill-rule="evenodd"></path><path d="M54 36.658C54 35.2713 55.1193 34.1472 56.5 34.1472 57.8807 34.1472 59 35.2713 59 36.658 59 38.0447 57.8807 39.1688 56.5 39.1688 55.1193 39.1688 54 38.0447 54 36.658Z" fill="#1D7EB3" fill-rule="evenodd"></path><path d="M92 39.671C92 38.2843 92.8954 37.1602 94 37.1602 95.1046 37.1602 96 38.2843 96 39.671 96 41.0577 95.1046 42.1818 94 42.1818 92.8954 42.1818 92 41.0577 92 39.671Z" fill="#D0602C" fill-rule="evenodd"></path><path d="M78 92.3983C78 91.2889 79.1193 90.3896 80.5 90.3896 81.8807 90.3896 83 91.2889 83 92.3983 83 93.5076 81.8807 94.4069 80.5 94.4069 79.1193 94.4069 78 93.5076 78 92.3983Z" fill-rule="evenodd"></path><path d="M38.5001 42.684 70.3743 67.4134" stroke="#D0602C" stroke-width="3.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><path d="M28.5001 66.7879 67.8141 69.5014" stroke="#D0602C" stroke-width="3.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><path d="M0 0 43.7032 13.7967" stroke="#1D7EB3" stroke-width="3.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(1 0 0 -1.00433 58.5001 89.6834)"></path><path d="M0 0 12.3103 33.7458" stroke="#000000" stroke-width="3.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd" transform="matrix(-1 0 0 1.00433 85.8104 30.6321)"></path><path d="M76.5001 32.6407 100.492 70.6506" stroke="#000000" stroke-width="3.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><path d="M34.5001 42.684 51.5087 86.0473" stroke="#D0602C" stroke-width="3.33333" stroke-miterlimit="8" fill="none" fill-rule="evenodd"></path><text font-family="Calisto MT,Calisto MT_MSFontService,sans-serif" font-weight="400" font-size="21" transform="matrix(1 0 0 1.00433 129.312 73)">BIN</text></g></svg>
        
      </span>
    
    
      <span class="title-text" data-tooltip="Home">
        
          <span class="title">Bin Zhang's Lab</span>
        
        
      </span>
    
  </a>

  <input class="nav-toggle" type="checkbox" aria-label="show/hide nav">

  <nav>
    
    
      
        <a href="/research/" data-tooltip="Learn about our research">
          Research
        </a>
      
    
      
        <a href="/publications/" data-tooltip="Published works">
          Publications
        </a>
      
    
      
        <a href="/team/" data-tooltip="About our team">
          Team
        </a>
      
    
      
        <a href="/joinus/" data-tooltip="Join our team">
          Join us!
        </a>
      
    
      
        <a href="/contact/" data-tooltip="Email, address, and location">
          Contact
        </a>
      
    
  </nav>
</header>

    <main>
      <!--
  modify main content of page:
  - add section breaks
  - attach section properties
  - filter out blank sections
-->






  
  
  

  <section class="background" data-size="page">
    <h1 id="publications">
<i class="icon fa-solid fa-microscope"></i>Publications</h1>

<p>Looking ahead, our research aims to develop interpretable biological foundation models, with a focus on antigen presentation and RNA isoform regulation at single-cell resolution. Moving beyond traditional gene-centric approaches, we explore isoform-level expression as a fundamental representation to characterize cell types, cellular states, and their spatiotemporal dynamics. This perspective enables a more precise and mechanistic understanding of cellular regulation.</p>

<p>By integrating multi-omics data, single-cell technologies, and AI-driven modeling, we seek to construct RNA-centric gene regulatory networks that address key limitations of existing biological foundation models, particularly in interpretability and regulatory resolution. Ultimately, our work aims to advance the quantitative understanding of living systems and to support the rational design of biological processes for applications in biomedicine and beyond.</p>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  <background></background>
  <dark></dark>
  <size></size>
-->

<h2 id="highlighted">Highlighted</h2>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd4d" class="citation-image" aria-label="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512202332549.png" alt="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd4d" class="citation-title">
        Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing
      </a>

      <div class="citation-authors" tabindex="0">
        Bin Zhang, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Genetics</span>
         · 
        <span class="citation-date">04 May 2023</span>
         · 
        <span class="citation-id">doi:10.1038/s41588-023-01363-5</span>
      </div>

      
        
          <div class="citation-description">
            Aberrant RNA splicing events resulting from DNA variations are common causes of genetic disorders. Two studies published in Nature Genetics independently describe methods to decipher DNA-variant-associated aberrant splicing using high-throughput RNA sequencing data.

          </div>
        

        

        
      
    </div>
  </div>
</div>
  </section>

  
  
  

  <section class="background" data-size="page">
    <!--
  <background></background>
  <dark></dark>
  <size></size>
-->

<h2 id="all">All</h2>

<div class="search-box">
  <input type="text" class="search-input" oninput="onSearchInput(this)" placeholder="Search items on this page">
  <button disabled data-tooltip="Clear search" aria-label="clear search" onclick="onSearchClear()">
    <i class="icon fa-solid fa-magnifying-glass"></i>
  </button>
</div>

<div class="search-info"></div>

<h3 id="2025">2025</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd3m" class="citation-image" aria-label="Stabilizing a mammalian RNA thermometer confers neuroprotection in subarachnoid hemorrhage">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512261711868.png" alt="Stabilizing a mammalian RNA thermometer confers neuroprotection in subarachnoid hemorrhage" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd3m" class="citation-title">
        Stabilizing a mammalian RNA thermometer confers neuroprotection in subarachnoid hemorrhage
      </a>

      <div class="citation-authors" data-tooltip="Min Zhang, Bin Zhang, Chengli Liu, Marco Preußner, Megha Ayachit, Weiming Li, Yafei Huang, Deyi Liu, Quanwei He, Ann-Kathrin Emmerichs, Stefan Meinke, Shu Chen, Lin Wang, Liduan Zheng, Qiubai Li, Qin Huang, Tom Haltenhof, Ruoxi Gao, Xianan Qin, Aifang Cheng, Tianzi Wei, Li Yu, Mario Schubert, Xin Gao, Mingchang Li, Florian Heyd" tabindex="0">
        Min Zhang, Bin Zhang, Chengli Liu, Marco Preußner, Megha Ayachit, …, Li Yu, Mario Schubert, Xin Gao, Mingchang Li, Florian Heyd

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Communications</span>
         · 
        <span class="citation-date">18 Sep 2025</span>
         · 
        <span class="citation-id">doi:10.1038/s41467-025-63911-3</span>
      </div>

      
        
          <div class="citation-description">
            Mammals tightly regulate their core body temperature, yet how cells sense and respond to small temperature changes remains incompletely understood. Here, we discover RNA G-quadruplexes (rG4s) as key thermosensors enriched near splice sites of cold-repressed exons. These thermosensing RNA structures, when stabilized, mask splice sites, reducing exon inclusion.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2024">2024</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g8qqnw" class="citation-image" aria-label="An AI Agent for Fully Automated Multi Omic Analyses">
        <img src="https://advanced.onlinelibrary.wiley.com/cms/asset/e7932ca7-a507-4ccc-97ce-2fbc2a59533a/advs9394-fig-0001-m.jpg" alt="An AI Agent for Fully Automated Multi Omic Analyses" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g8qqnw" class="citation-title">
        An AI Agent for Fully Automated Multi‐Omic Analyses
      </a>

      <div class="citation-authors" data-tooltip="Juexiao Zhou, Bin Zhang, Guowei Li, Xiuying Chen, Haoyang Li, Xiaopeng Xu, Siyuan Chen, Wenjia He, Chencheng Xu, Liwei Liu, Xin Gao" tabindex="0">
        Juexiao Zhou, Bin Zhang, Guowei Li, Xiuying Chen, Haoyang Li, …, Siyuan Chen, Wenjia He, Chencheng Xu, Liwei Liu, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Advanced Science</span>
         · 
        <span class="citation-date">03 Oct 2024</span>
         · 
        <span class="citation-id">doi:10.1002/advs.202407094</span>
      </div>

      
        
          <div class="citation-description">
            With the fast-growing and evolving omics data, the demand for streamlined and adaptable tools to handle bioinformatics analysis continues to grow. In response to this need, Automated Bioinformatics Analysis (AutoBA) is introduced, an autonomous AI agent designed explicitly for fully automated multi-omic analyses based on large language models (LLMs).

          </div>
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd3s" class="citation-image" aria-label="An isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512261710900.png" alt="An isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd3s" class="citation-title">
        An isoform-resolution transcriptomic atlas of colorectal cancer from long-read single-cell sequencing
      </a>

      <div class="citation-authors" data-tooltip="Zhongxiao Li, Bin Zhang, Jia Jia Chan, Hossein Tabatabaeian, Qing Yun Tong, Xiao Hong Chew, Xiaonan Fan, Patrick Driguez, Charlene Chan, Faith Cheong, Shi Wang, Bei En Siew, Ian Jse-Wei Tan, Kai-Yin Lee, Bettina Lieske, Wai-Kit Cheong, Dennis Kappei, Ker-Kan Tan, Xin Gao, Yvonne Tay" tabindex="0">
        Zhongxiao Li, Bin Zhang, Jia Jia Chan, Hossein Tabatabaeian, Qing Yun Tong, …, Wai-Kit Cheong, Dennis Kappei, Ker-Kan Tan, Xin Gao, Yvonne Tay

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Cell Genomics</span>
         · 
        <span class="citation-date">11 Sep 2024</span>
         · 
        <span class="citation-id">doi:10.1016/j.xgen.2024.100641</span>
      </div>

      
        
          <div class="citation-description">
            Colorectal cancer (CRC) ranks as the second leading cause of cancer deaths globally. In recent years, short-read single-cell RNA sequencing (scRNA-seq) has been instrumental in deciphering tumor heterogeneities. However, these studies only enable gene-level quantification but neglect alterations in transcript structures arising from alternative end processing or splicing. In this study, we integrated short- and long-read scRNA-seq of CRC samples to build an isoform-resolution CRC transcriptomic atlas.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2023">2023</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd4d" class="citation-image" aria-label="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512202332549.png" alt="Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd4d" class="citation-title">
        Deciphering DNA variant-associated aberrant splicing with the aid of RNA sequencing
      </a>

      <div class="citation-authors" tabindex="0">
        Bin Zhang, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Genetics</span>
         · 
        <span class="citation-date">04 May 2023</span>
         · 
        <span class="citation-id">doi:10.1038/s41588-023-01363-5</span>
      </div>

      
        
          <div class="citation-description">
            Aberrant RNA splicing events resulting from DNA variations are common causes of genetic disorders. Two studies published in Nature Genetics independently describe methods to decipher DNA-variant-associated aberrant splicing using high-throughput RNA sequencing data.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/hbgd36" class="citation-image" aria-label="Accurate transcriptome-wide identification and quantification of alternative polyadenylation from RNA-seq data with APAIQ">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512261715679.png" alt="Accurate transcriptome-wide identification and quantification of alternative polyadenylation from RNA-seq data with APAIQ" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/hbgd36" class="citation-title">
        Accurate transcriptome-wide identification and quantification of alternative polyadenylation from RNA-seq data with APAIQ
      </a>

      <div class="citation-authors" data-tooltip="Yongkang Long, Bin Zhang, Shuye Tian, Jia Jia Chan, Juexiao Zhou, Zhongxiao Li, Yisheng Li, Zheng An, Xingyu Liao, Yu Wang, Shiwei Sun, Ying Xu, Yvonne Tay, Wei Chen, Xin Gao" tabindex="0">
        Yongkang Long, Bin Zhang, Shuye Tian, Jia Jia Chan, Juexiao Zhou, …, Shiwei Sun, Ying Xu, Yvonne Tay, Wei Chen, Xin Gao

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Genome Research</span>
         · 
        <span class="citation-date">28 Feb 2023</span>
         · 
        <span class="citation-id">doi:10.1101/gr.277177.122</span>
      </div>

      
        
          <div class="citation-description">
            Alternative polyadenylation (APA) enables a gene to generate multiple transcripts with different 3′ ends, which is dynamic across different cell types or conditions. Many computational methods have been developed to characterize sample-specific APA using the corresponding RNA-seq data, but suffered from high error rate on both polyadenylation site (PAS) identification and quantification of PAS usage (PAU), and bias toward 3′ untranslated regions. Here we developed a tool for APA identification and quantification (APAIQ) from RNA-seq data, which can accurately identify PAS and quantify PAU in a transcriptome-wide manner.

          </div>
        

        

        
      
    </div>
  </div>
</div>

<h3 id="2022">2022</h3>

<div class="citation-container">
  <div class="citation">
    
      <a href="https://doi.org/g8p6ct" class="citation-image" aria-label="Pan-cancer pervasive upregulation of 3 UTR splicing drives tumourigenesis">
        <img src="https://blog-img-1304093596.cos.ap-shanghai.myqcloud.com/202512202333121.png" alt="Pan-cancer pervasive upregulation of 3 UTR splicing drives tumourigenesis" loading="lazy" onerror="this.src = '/images/fallback.svg'; this.onerror = null;">
      </a>
    

    <div class="citation-text">
      
      <i class="icon fa-solid fa-scroll"></i>

      <a href="https://doi.org/g8p6ct" class="citation-title">
        Pan-cancer pervasive upregulation of 3′ UTR splicing drives tumourigenesis
      </a>

      <div class="citation-authors" data-tooltip="Jia Jia Chan, Bin Zhang, Xiao Hong Chew, Adil Salhi, Zhi Hao Kwok, Chun You Lim, Ng Desi, Nagavidya Subramaniam, Angela Siemens, Tyas Kinanti, Shane Ong, Avencia Sanchez-Mejias, Phuong Thao Ly, Omer An, Raghav Sundar, Xiaonan Fan, Shi Wang, Bei En Siew, Kuok Chung Lee, Choon Seng Chong, Bettina Lieske, Wai-Kit Cheong, Yufen Goh, Wee Nih Fam, Melissa G. Ooi, Bryan T. H. Koh, Shridhar Ganpathi Iyer, Wen Huan Ling, Jianbin Chen, Boon-Koon Yoong, Rawisak Chanwat, Glenn Kunnath Bonney, Brian K. P. Goh, Weiwei Zhai, Melissa J. Fullwood, Wilson Wang, Ker-Kan Tan, Wee Joo Chng, Yock Young Dan, Jason J. Pitt, Xavier Roca, Ernesto Guccione, Leah A. Vardy, Leilei Chen, Xin Gao, Pierce K. H. Chow, Henry Yang, Yvonne Tay" tabindex="0">
        Jia Jia Chan, Bin Zhang, Xiao Hong Chew, Adil Salhi, Zhi Hao Kwok, …, Leilei Chen, Xin Gao, Pierce K. H. Chow, Henry Yang, Yvonne Tay

      </div>

      <div class="citation-details">
        <span class="citation-publisher">Nature Cell Biology</span>
         · 
        <span class="citation-date">26 May 2022</span>
         · 
        <span class="citation-id">doi:10.1038/s41556-022-00913-z</span>
      </div>

      
        
          <div class="citation-description">
            Most mammalian genes generate messenger RNAs with variable untranslated regions (UTRs) that are important post-transcriptional regulators. In cancer, shortening at 3′ UTR ends via alternative polyadenylation can activate oncogenes. However, internal 3′ UTR splicing remains poorly understood as splicing studies have traditionally focused on protein-coding alterations.

          </div>
        

        

        
      
    </div>
  </div>
</div>
  </section>


    </main>
    


<footer class="background" style="--image: url('/images/background.png')" data-dark="true" data-size="wide">
  <!--
    <div>
      Extra details like contact info or address
    </div>
  -->

  <div>
    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="mailto:bin.zhang@mbzuai.ac.ae" data-tooltip="Email" data-style="bare" aria-label="Email">
      <i class="icon fa-solid fa-envelope"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://orcid.org/0000-0001-8835-8370" data-tooltip="ORCID" data-style="bare" aria-label="ORCID">
      <i class="icon fa-brands fa-orcid"></i>
      
    </a>
  </div>


    
      
      
      



  <div class="button-wrapper">
    <a class="button" href="https://scholar.google.com/citations?user=lFl6evoAAAAJ" data-tooltip="Google Scholar" data-style="bare" aria-label="Google Scholar">
      <i class="icon fa-brands fa-google"></i>
      
    </a>
  </div>


    
  </div>

  <div>
    © 2025
    Bin Zhang's Lab
      |   Built with
    <a href="https://github.com/greenelab/lab-website-template">
      Lab Website Template
    </a>
  </div>

  <input type="checkbox" class="dark-toggle" data-tooltip="Dark mode" aria-label="toggle dark mode" oninput="onDarkToggleChange(event)">
</footer>

  </body>
</html>
